jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/Dispatcher.java

```diff
@@ -620,19 +619,30 @@ public DispatcherType getDispatcherType()
         @Override
         public String getPathInfo()
         {
+            if (_servletPathMapping == null)
+                return super.getPathInfo();
             return _servletPathMapping.getPathInfo();
         }
 
         @Override
         public String getServletPath()
         {
+            if (_servletPathMapping == null)
+                return super.getServletPath();
             return _servletPathMapping.getServletPath();
         }
 
+        @Override
+        public HttpServletMapping getHttpServletMapping()
+        {
+            if (_servletPathMapping == null)
```

Comment:
For the same method in `ForwardRequest` this check is `_named != null`, maybe these should be consistent.

Reply:
If this is `_named != null` then it fails some DispatcherTests. (specifically AsyncContext.dispatch() tests to destination paths that don't exist and should throw a 404)
If this is `_servletPathMapping == null` then it passes both ErrorPageTest and DispatcherTest.

Reply:
I tried to create a test case that uses a named servlet reference to handle a `sendError(597)` but was unable to support it.

``` xml
	<error-page>
		<error-code>597</error-code>
		<location>NamedErrorServlet</location>
	</error-page>
```

I don't think supporting `_named` in `Dispatcher$ErrorRequest` is possible.

Reply:
I do not think it is possible to get a named error dispatcher as we only ever lookup a dispatcher for an error page by URI.  

Could probably put an `assert _named==null` in the `error` method.

Reply:
> Could probably put an `assert _named==null` in the `error` method.

Done.